dotCMS/src/integration-test/java/com/dotcms/content/elasticsearch/business/ESIndexUtilTest.java

```diff
@@ -0,0 +1,199 @@
+import org.elasticsearch.client.RequestOptions;
+import org.elasticsearch.common.settings.Settings;
+import org.junit.BeforeClass;
+import com.dotcms.util.IntegrationTestInitService;
+import org.junit.Test;
+
+import static org.jgroups.util.Util.assertEquals;
+
+public class ESIndexUtilTest {
+
+    @BeforeClass
+    public static void prepare () throws Exception {
+        //Setting web app environment
+        IntegrationTestInitService.getInstance().init();
+    }
+
+    /**
+     * Method to Test: {@link ESIndexUtil#isReadOnly(String...)}
+     * When: The Index is not read only
+     * Should:return false
+     *
+     * @throws DotDataException
+     */
+    @Test
+    public void shouldReturnFalseIfTheCurrentIndicesAreReadOnly() throws DotDataException {
+        final IndiciesInfo indiciesInfo = APILocator.getIndiciesAPI().loadIndicies();
+
+        try {
+            putReadOnly(indiciesInfo.getWorking(), false);
+
+            final boolean readOnly = ESIndexUtil.isReadOnly(indiciesInfo.getWorking());
+
+            assertEquals(false, readOnly);
+        } finally {
+            putReadOnly(indiciesInfo.getWorking(), false);
+        }
+    }
+
+    /**
+     * Method to Test: {@link ESIndexUtil#isReadOnly(String...)}
+     * When: The Index is read only
+     * Should:return true
+     *
+     * @throws DotDataException
+     */
+    @Test
+    public void shouldReturnTrueIfTheCurrentIndicesAreReadOnly() throws DotDataException {
+        final IndiciesInfo indiciesInfo = APILocator.getIndiciesAPI().loadIndicies();
+
+        try {
+            putReadOnly(indiciesInfo.getWorking(), true);
+
+            final boolean readOnly = ESIndexUtil.isReadOnly(indiciesInfo.getWorking());
+
+            assertEquals(true, readOnly);
+        } finally {
+            putReadOnly(indiciesInfo.getWorking(), false);
+        }
+    }
+
+    /**
+     * Method to Test: {@link ESIndexUtil#isReadOnly(String...)}
+     * When: The Index not exists
+     * Should: throw a {@link org.elasticsearch.ElasticsearchStatusException}
+     *
+     * @throws DotDataException
+     */
+    @Test (expected = ElasticsearchStatusException.class)
+    public void whenTheIndexNotExistsShouldThrowException() {
+        final boolean readOnly = ESIndexUtil.isReadOnly("not_Exists");
+        assertEquals(true, readOnly);
+    }
+
+    /**
+     * Method to Test: {@link ESIndexUtil#putReadOnlyToFalse(String...)}
+     * When: The index exists
+     * Should: should set it to read only
+     *
+     * @throws DotDataException
+     */
+    @Test
+    public void shouldPutReadonlyInTrue() throws DotDataException {
+        final IndiciesInfo indiciesInfo = APILocator.getIndiciesAPI().loadIndicies();
+
+        try {
+            ESIndexUtil.putReadOnlyToFalse(indiciesInfo.getWorking());
+
+            final boolean readOnly = ESIndexUtil.isReadOnly(indiciesInfo.getWorking());
+
+            assertEquals(true, !readOnly);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use assertTrue(x)/assertFalse(x) instead of assertEquals(true, x)/assertEquals(false, x) or assertEquals(Boolean.TRUE, x)/assertEquals(Boolean.FALSE, x).](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5642172)